Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.12-slim
+LABEL authors="kates1999"
+
+WORKDIR /py-docker-weather-api
+COPY . .
+
+RUN pip install requests
```

Comment:
Consider using a `requirements.txt` file to manage dependencies. This will make it easier to handle multiple dependencies and ensure consistency across environments. You can then use `RUN pip install -r requirements.txt` to install them.